jobfunnel/backend/scrapers/glassdoor.py

```diff
@@ -352,3 +352,26 @@ def quantize_radius(self, radius: int) -> int:
         elif radius >= 100:
             radius = 100
         return GLASSDOOR_RADIUS_MAP[radius]
+
+
+class GlassDoorScraperUKEng(BaseGlassDoorScraper, BaseUKEngScraper):
+
+    def quantize_radius(self, radius: int) -> int:
+        """Get a UK raduius (km)
+        FIXME: use numpy.digitize instead
+        """
+        if radius < 10:
```

Comment:
since canada and uk are performing the same conversion, can we share the quantize_radius method (perhaps we could move it to a method?)

Reply:
I'm thinking of creating a new class MonsterImperialRadius so that _convert_radius can be shared by MonsterScraperCANEng and MonsterScraperUKEng. 

Likewise GlassDoorImperialRadius for sharing quantize_radius between GlassDoorScraperCANEng and GlassDoorScraperUKEng

Reply:
I like this plan, I think this makes sense in the context of localisation with shared radius calcs.

Reply:
ok great! For Monster, I've implemented a MonsterMetricRadius, which is inherited by MonsterScraperCANEng and MonsterScraperUKEng and likewise for Glassdoor.

The order of ie. MonsterScraperUKEng(MonsterMetricRadius, BaseMonsterScraper,BaseUKEngScraper) as opposed to MonsterScraperUKEng(BaseMonsterScraper, BaseUKEngScraper, MonsterMetricRadius) is preventing a break in baseclass conflicting with the @abstractmethod.